src/dev.config.ts

```diff
@@ -123,7 +114,7 @@ window['${libraryName}'].base = '${base}'</script>`,
 		config.plugins.push(
 			new BuildTimeRender({
 				...args['build-time-render'],
-				sync: singleBundle,
+				sync: true,
```

Comment:
Does BTR always need to be "sync" with these changes?

Reply:
I'm not positive whether BTR breaks if it's async for a single bundle. Since the dev build here is now always a single bundle I just converted all the single bundle conditionals to `true`.

Reply:
BTR is still async unless `singleBundle` is explicitly set to true for the dist build, if that wasn't clear.